PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+# Pull Request Description
+- KEP-< ISSUE NUMBER >: < KEP NAME >
+- One-line PR description: <!-- short description of work done in PR e.g. updating milestone, adding new KEP, adding test requirements… -->
+
+_Please use the following format when naming your PR: **< Issue Number >:< Issue Description >** (e.g. KEP-000: adding beta graduation criteria)_
```

Comment:
I'm not sure if we want to make this visible or make it a comment. what do you think @kikisdeliveryservice?

Reply:
I think make it a comment